docs/rules/no-restricted-static-attribute.md

```diff
@@ -43,6 +43,11 @@ Alternatively, the rule also accepts objects.
     {
       "key": "stlye",
       "message": "Using \"stlye\" is not allowed. Use \"style\" instead."
+    },
+    {
+      "key": "style",
+      "element": /^([A-Z$][a-zA-Z_$]*)$/,
```

Comment:
Shouldn't it be a string? Also, the `$` signs don't make sense here:

```suggestion
      "element": "/^([A-Z][a-zA-Z_-]*)$/",
```

Reply:
oh sorry, I made a new commit which revised this typo. Thank you for your review!